aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -406,6 +429,10 @@ class QConvInt8 final {
 };
 
 TORCH_LIBRARY_IMPL(quantized, QuantizedCUDA, m) {
+  m.impl(TORCH_SELECTIVE_NAME("quantized::conv1d"), QConv1dInt8<false>::run);
+  m.impl(TORCH_SELECTIVE_NAME("quantized::conv1d_relu"), QConv1dInt8<true>::run);
```

Comment:
maybe only register `quantized::conv1d.new` and `quantized::conv1d_relu.new`? I think we want to deprecate the rest

Reply:
I remember doing this and it seems it was causing the issues with the dispatcher not being able to find the QuantizedCuda backend for conv1d.

Reply:
ah I see. quantized cpu doesn't have .new. so we should just keep `quantized::conv1d` and get rid of `quantized::conv1d.new`

Reply:
`quantized::conv1d.new` is the new one, I think we should get rid of `quantized::conv1d` and just keep `quantized::conv1d.new`

Reply:
yes, but what I'm saying is currnetly QuantizedCPU doesn't use `quantized::conv1d.new`. it only defines `quantized::conv1d` and it uses the packed weight signature (see https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/library.cpp#L64-L65). it's not consistent with conv2d. 

so maybe we can just change  `quantized::conv1d` in QuantizedCPU to `quantized::conv1d.new` so we're consistent with conv2d?